rest-api/dao/physical_measurements_dao.py

```diff
@@ -328,6 +329,19 @@ def make_measurement_id(physical_measurements_id, measurement_count):
     return (physical_measurements_id * 1000) + measurement_count
 
   @staticmethod
+  def get_preferred_coding(code):
```

Comment:
Would call the arg `codeable_concept` (or `concept`), as in `get_preferred_coding(concept)` -- just to avoid having to say "a code has codings which have codes". (I know that some fields in FHIR are named `code`, like `Observation.code` -- but the *type* of this field is `CodeableConcept`.)

Reply:
Done.